src/app/rundown/components/segment/segment.component.html

```diff
@@ -1,4 +1,44 @@
-<div class="segment" [ngClass]="{'segment-on-air': segment.isOnAir}">
+<div class="c-segment" [class.c-segment--on-air]="segment.isOnAir" [class.c-segment--set-as-next]="segment.isNext">
+  <div class="c-segment__header">
+    <div class="c-segment__header-toolbar">
+      <span>123</span>
+      <span style="margin-left: 50px;">00:30</span>
+    </div>
+    <div class="c-segment__title">
+      {{ segment.name }}
+    </div>
+
+  </div>
+
+  <div class="c-segment__controls">
+    <div class="c-segment__controls-toolbar">Hello</div>
+    <div class="c-segment__piece-layers">
+      <div class="c-segment__piece-layer" *ngFor="let pieceLayer of pieceLayers">
+        {{ pieceLayer }}
+      </div>
+    </div>
+  </div>
```

Comment:
Do all these need to be divs?
Seems like `<div class="c-segment__controls-toolbar">Hello</div>` could be `<span class="c-segment__controls-toolbar">Hello</span>`.
and `<div class="c-segment__piece-layers">` seems to do nothing, so why is it there?

There seem to be a tendency to use way too many divs in general.

Reply:
`.c-segment__controls-toolbar` should be a blocking element and will contain zoom/scale controls.
`.c-segment__piece-layers` is there to regulate the space where the layers are confined in the same way as `ol` or `ul` do for `li`.